scss/_mixins.scss

```diff
@@ -61,6 +61,10 @@
 	.o-topper--news-package {
 		@include oTopperThemeNewsPackage;
 	}
+
+	.o-topper--right-rail {
+		@include oTopperThemeRightRail;
```

Comment:
Not related - but there's a `o-topper--centered` which for some reason is in `oTopperElement`. Could we make that a theme too? (it's effectively the same sort of thing, a different grid column setup)

Reply:
sure